yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -52,8 +53,8 @@
        <b>Uploaded At: </b>{{version.last_modified}}    
     </div>
 
-    <div class = title>
-        Runs For Choosen Version:
+    <div class = run-text>
```

Comment:
I'm not going to make you fix this, but in general you see html formatted like `class=title` instead of `class = title`. 
Just an FYI so we can try to keep things consistent.

Reply:
I'll make a note of this and fix it in the next PR.